guides/common/modules/proc_creating-a-local-source-for-a-custom-file-type-repository.adoc

```diff
@@ -0,0 +1,71 @@
+[id="Creating_a_Local_Source_for_a_Custom_File_Type_Repository_{context}"]
+= Creating a Local Source for a {customfiletypetitle} Repository
+
+You can create a {customfiletype} repository source, from a directory of files, on the base system where {Project} is installed using *Pulp Manifest*.
+You can then synchronize the files into {ProjectServer}.
+When you add files to a file type repository, you can work with the files as with any other repository.
```

Comment:
This is unclear to me. Do you mean "you can manage content of type *file* as any other content, such as yum and deb"?

Reply:
How about:
`You can then synchronize the files into a repository on {ProjectServer} and manage the {customfiletype} content like any other content type.`
Would that make it sound clearer?

Reply:
Yes, this is much clearer to me.